File: java/assembly/instructions/PopInt.java
package assembly.instructions;

//3AC to push a value on the stack

public class PopInt extends Instruction {
    
    public PopInt(String src) {
        super();
        this.src1 = src;
    }

    public String toString() {
        return "POPINT " + src1;
    }

    @Override
    public boolean is3AC() {
        return true;
    }
}

File: python/MicroCCompiler/assembly/instructions/PopInt.py
from .Instruction import Instruction, OpCode

class PopInt(Instruction):
  def __init__(self, src: str):
    super().__init__()
    self.src1 = src

  def __str__(self):
    return "POPINT " + self.src1

  def is3AC(self, o=None):
    return True
